### PR TITLE
update translation

### PR DESCRIPTION
**Pull request by lishaoxia1985**

### CODE REVIEW

File: core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt [hunk 1]
```diff
@@ -91,9 +94,9 @@ class BattleTable(val worldScreen: WorldScreen): Table() {
         add("{Strength}: ".tr()+attacker.getAttackingStrength())
         add("{Strength}: ".tr()+defender.getDefendingStrength()).row()
 
-        val attackerModifiers = BattleDamage().getAttackModifiers(attacker,defender)  .map { it.key.tr()+": "+(if(it.value>0)"+" else "")+(it.value*100).toInt()+"%" }
+        val attackerModifiers = BattleDamage().getAttackModifiers(attacker,defender).map { (if(it.key.startsWith("vs ")) ("vs ["+it.key.replace("vs ","")+"]").tr() else it.key.tr())+": "+(if(it.value>0)"+" else "")+(it.value*100).toInt()+"%" }
```
Comment (yairm210) [inline core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt]: This is not readable as one line...

File: core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt [hunk 2]
```diff
@@ -91,9 +94,9 @@ class BattleTable(val worldScreen: WorldScreen): Table() {
...
         val defenderModifiers = if (defender is MapUnitCombatant)
-                                    BattleDamage().getDefenceModifiers(attacker, defender).map { it.key.tr()+": "+(if(it.value>0)"+" else "")+(it.value*100).toInt()+"%" }
+                                    BattleDamage().getDefenceModifiers(attacker, defender).map { (if(it.key.startsWith("vs ")) ("vs ["+it.key.replace("vs ","")+"]").tr() else it.key.tr())+": "+(if(it.value>0)"+" else "")+(it.value*100).toInt()+"%" }
```
Comment (yairm210) [inline core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt]: Same

File: android/assets/jsons/Translations/Other.json [hunk 3]
```diff
@@ -1411,17 +1415,8 @@
 		Czech:"Zdraví města"
 }
 
-	"Captured":{ // Shown in the battle table when attacking a civilian unit or a defeated city
```
Comment (yairm210) [inline android/assets/jsons/Translations/Other.json]: Please migrate to 'captured!' entry

Comment (lishaoxia1985) [reply android/assets/jsons/Translations/Other.json]: you can see line 1441

File: core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt (hunk 1 above)
Comment (lishaoxia1985) [reply core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt]: i don't know how to solve it. it runs very well.
it means that:
for example, if it.key = "vs city" or "vs units in open terrain", we can use the method replace() to get the words "city" or "units in open terrain", at last we get the word "vs [city]" and "vs [units in open terrain]", i have translate the word "vs [unit]", so all words can be translated by tr().

Comment (yairm210) [reply core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt]: I have no problem with the logic, just with the readability.
Separate into separate lines

Comment (lishaoxia1985) [reply core/src/com/unciv/ui/worldscreen/bottombar/BattleTable.kt]: now it's ok like you wish.

**Pull request merged**